### ISSUE TITLE
[FastPitch/Pytorch] Multi-speaker - Which repo to use to train Tacotron 2 model[s]?

### ISSUE THREAD

**Issue by jmasterx** [NVIDIA/DeepLearningExamples#835]
Hi

I would like to train fastpitch, but the filelists expect durations, mels, pitch, etc extracted from a tacotron model. But how does this work in the multi speaker scenario? Is it expected that we train 1 tacotron 2 model per speaker, then for each of these models we extract the mels, duration, pitch, etc? Or is there a multi speaker tacotron 2 that should be used where we can learn on all the speakers at once? Do I train a mellotron model and that will be compatible ?

Thank you

**Comment by alancucki** [comment]
Hi @jmasterx ,

currently we do not provide pre-trained multi-speaker Tacotron2 nor Mellotron checkpoints.

To train a multi-speaker FastPitch model, you would need an external aligner (like Tacotron2 in the repository). This could be a multi-speaker model, or a collection of single-speaker models - it does not matter as long as all those models are converged enough to properly align. Using a collection of models would require some adjustments in the `extract_mels.py` file.

We're working towards automatic alignment methods, that would relieve from using Tacotron2 aligners without any loss of quality.

**Comment by jmasterx** [comment]
@alancucki So if I train my own Mellotron model from scratch and it gets good enough to have good alignments, could I use that in extract_mels.py to extract the alignments?

**Comment by jmasterx** [comment]
For anyone wondering, the Tacotron GST from Mellotron does not seem to be compatible with FastPitch. But the Tacotron 2 of this repo, which is single speaker, is compatible.
